source/javascripts/components/_Step.js

```diff
@@ -29,9 +29,31 @@ import { normalizeIconUrl } from "./StepItem/StepItem";
 				return this.id;
 			}
 
-			return this.cvs;
+			return diplayNameFromCvs(this.displayCvs());
 		};
 
+		Step.prototype.displayCvs = function() {
+			return (
+				(this.cvs.startsWith("git::") || this.cvs.startsWith("path::")) 
+					? this.cvs.replace(/(git|path)::/g, "")
+					: this.cvs
+			);
```

Comment:
can't we tackle this in a single regex? like replace `/^(git|path)::/g`, it will only replace this wen the cvs starts with that prefix. You should also add unit tests for this :)

Reply:
Will do.